### DISCUSSION TITLE
2024.6.0

### DISCUSSION THREAD

**Discussion by AndrewBastin** [hoppscotch/hoppscotch#4154] [Announcements]
This is a major release with new features like Personal Access Tokens, updates to the Admin Dashboard, updates to backend hosting requirements and better integration with the Hoppscotch CLI.

## What's Changed
* HSB-445 feature: storing user last login timestamp by @mirarifhasan in https://github.com/hoppscotch/hoppscotch/pull/4074
* feat: adding support for `hopp-cli` in self-host Hoppscotch by @balub in https://github.com/hoppscotch/hoppscotch/pull/4079
* chore: changed the returned status type for errors in the access-tokens by @balub in https://github.com/hoppscotch/hoppscotch/pull/4107
* HSB-450 feature: user last active on by @mirarifhasan in https://github.com/hoppscotch/hoppscotch/pull/4121
* fix: large content scroll bug in codemirror editor by @nivedin in https://github.com/hoppscotch/hoppscotch/pull/4138
* feat: introduce personal access tokens for authorization by @jamesgeorge007 in https://github.com/hoppscotch/hoppscotch/pull/4094
* hotix: prevent unwanted variable wrapping during Insomnia import by @anwarulislam in https://github.com/hoppscotch/hoppscotch/pull/4076
* feat: added codegen tab in the sidbear by @anwarulislam in https://github.com/hoppscotch/hoppscotch/pull/4099
* fix: styling issue on selection in codemirror by @nivedin in https://github.com/hoppscotch/hoppscotch/pull/4122
* refactor(common): improvements to banner handling logic by @JoelJacobStephen in https://github.com/hoppscotch/hoppscotch/pull/4123
* HSB-452 hotfix: storing user in local variable by @mirarifhasan in https://github.com/hoppscotch/hoppscotch/pull/4146
* feat(sh-admin): introducing last active status to users table and individual user's page in dashboard  by @JoelJacobStephen in https://github.com/hoppscotch/hoppscotch/pull/4077
* HSB-439 feature: invite link with SMTP optional by @mirarifhasan in https://github.com/hoppscotch/hoppscotch/pull/4078
* feat(cli): access team workspace collections and environments by @jamesgeorge007 in https://github.com/hoppscotch/hoppscotch/pull/4095
* feat: CLI collection runner command generation UI flow by @jamesgeorge007 in https://github.com/hoppscotch/hoppscotch/pull/4141
* feat: added change log prompt for PWA updates by @anwarulislam in https://github.com/hoppscotch/hoppscotch/pull/4098
* feat(sh-admin): introducing advanced SMTP configurations and invite links to dashboard by @JoelJacobStephen in https://github.com/hoppscotch/hoppscotch/pull/4087
* refactor(sh-admin): improvements to invite links in dashboard by @JoelJacobStephen in https://github.com/hoppscotch/hoppscotch/pull/4152
* feat: copyable invite links by @amk-dev in https://github.com/hoppscotch/hoppscotch/pull/4153

**Full Changelog**: https://github.com/hoppscotch/hoppscotch/compare/2024.3.4...2024.6.0

<hr /><em>This discussion was created from the release <a href='https://github.com/hoppscotch/hoppscotch/releases/tag/2024.6.0'>2024.6.0</a>.</em>

**Comment by jubu02** [comment]
I'm no longer get the response body displayed in my widgets, I can download the response but not browse it - why is that? Can't find any settings related to this new behaviour?

**Reply by AndrewBastin** [reply]
@jubu02 can you open an issue with more info along with some screenshots of the issue ?
